libraries/radpdfprocessing/overview.md

```diff
@@ -10,49 +10,33 @@ position: 0
 
 # Overview
 
-__RadPdfProcessing__ is a processing library that allows creation, import and export of PDF documents from your code.
-![Rad Pdf Processing Overview 01](images/RadPdfProcessing_Overview_WPF_01.PNG)
+RadPdfProcessing is a processing library that allows you to create, import and export PDF documents from your code. You can use it in any web or desktop .NET application without relying on third-party software like  Adobe Acrobat.
 
->noteIf you still don't have **Telerik Document Processing installed**, check the **[First Steps]({%slug getting-started-first-steps%})** topic to learn how you can obtain the packages through the different suites. 
+In this article, we list the library's most popular features. If you want to learn how to use the library, head straight to the **[Getting Started with RadPdfProcessing]({%slug radpdfprocessing-getting-started%})** article.
```

Comment:
I suggest moving the link to the Getting Started article to the top of the Overview article. This is to help readers that don't want to go through an overview article. Currently, there are multiple notes at the begging of the article, and the reader may skip this link unintentionally.
I did this change to other Overview articles in this PR as well.